app/lists.rb

```diff
@@ -30,8 +33,10 @@ module Listing
 
   command :wingme, description: "Adds you to the wing queue" do |event|
   #Adds the person who used the command to wing_list.
-    break if event.channel.name.downcase == 'freetalk'
-	# Forces people to not use freetalk for wing requests.
+    break if wing_switch == false
```

Comment:
How does this ever get set to false? If it's a switch I don't want to be editing the code to change it. You should add something like `&disablewings` or something to flip it on/off.